Formula/tnftp.rb

```diff
@@ -0,0 +1,38 @@
+class Tnftp < Formula
+  desc "NetBSD's FTP client (built from macOS Sierra sources)"
+  homepage "https://opensource.apple.com/"
+  url "https://opensource.apple.com/tarballs/lukemftp/lukemftp-16.tar.gz"
+  sha256 "ba35a8e3c2e524e5772e729f592ac0978f9027da2433753736e1eb1f1351ae9d"
+
+  depends_on :xcode => :build
+
+  conflicts_with "inetutils", because: "both install `ftp' binaries"
+
+  def install
+    # Trying to use Apple's pre-supplied Makefile resulted
+    # in headaches... they have made the build process
+    # specifically for installing to /usr/bin and so it
+    # just doesn't play well with homebrew.
+
+    # so just build straight from ftp's own sources
+    Dir.chdir "tnftp"
+
+    system "./configure"
+    system "make", "all"
+    system "strip", "-x", "src/ftp" # this is done in Apple's `post-install' target
+
+    bin.install "src/ftp"
+    man1.install "src/ftp.1"
+  end
+
+  test do
```

Comment:
Anything referencing a formula should use the `bin` path, even inside said formula. So `bin/"ftp"`, not simply "ftp".

Reply:
I'm not sure how I'm "referencing a formula" or what that means.

Reply:
Er, not sure how to explain it really. Essentially you should use `system bin/"xyz"` always when referring to Homebrew formulae in `test do`. So for other formula you'd use something like:

```ruby
system Formula["openssl"].opt_bin/"openssl", "blah"
```
for example. And for self-referencing, where you're using executables from the created formula, you use the short syntax:
```
system bin/"ftp", "blah"
```

Reply:
@DomT4 Ok, well I've set everything to use `#{bin}/ftp`. Is that sufficient?